exchange/utils.go

```diff
@@ -362,6 +363,33 @@ func buildRequestExtAlternateBidderCodes(bidder string, accABC *openrtb_ext.ExtA
 	return nil
 }
 
+func buildRequestExtMultiBid(adapter string, reqMultiBid []*openrtb_ext.ExtMultiBid, adapterABC *openrtb_ext.ExtAlternateBidderCodes) []*openrtb_ext.ExtMultiBid {
+	// TODO: send multiBid to adapters under adapterABC
```

Comment:
Is this TODO comment still needed?

Reply:
added code to address extra-bid feature with multibid along with its test cases